src/test/java/copy_credentials/CopyCredentialsInteractorTest.java

```diff
@@ -0,0 +1,52 @@
+package copy_credentials;
+
+import java.awt.Toolkit;
+import java.awt.datatransfer.DataFlavor;
+import java.awt.datatransfer.UnsupportedFlavorException;
+import java.io.IOException;
+
+import org.junit.jupiter.api.Test;
+
+import service.copy_credentials.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CopyCredentialsInteractorTest {
+
+    @Test
+    public void testCopyUsername() throws IOException, UnsupportedFlavorException {
+        UsernameInputData usernameInputData = new UsernameInputData("Amaan");
+        CopyCredenentialsOutputBoundary copyCredenentialsOutputBoundary = new CopyCredenentialsOutputBoundary() {
+            @Override
+            public void displayUsernameCopyMessage(String message) {
+
+            }
+
+            @Override
+            public void displayPasswordCopyMessage(String message) {
+
+            }
+        };
+        CopyCredentialsInputBoundary copyCredentialsInputBoundary = new CopyCredentialsInteractor(copyCredenentialsOutputBoundary);
+        copyCredentialsInputBoundary.copyUsername(usernameInputData);
+        assertEquals(Toolkit.getDefaultToolkit().getSystemClipboard().getData(DataFlavor.stringFlavor), "Amaan");
```

Comment:
Then use testUsername here. That way you can change it easily without having to search for all appearances.